client/src/document/ingredients/browser-compatibility-table/legend.tsx

```diff
@@ -11,8 +11,8 @@ const LEGEND_LABELS = {
   no: "No support",
   unknown: "Compatibility unknown",
   experimental: "Experimental. Expect behavior to change in the future.",
-  "note-warning": "Non-standard. Check cross-browser support before using.",
-  "thumbs-down": "Deprecated. Not for use in new websites.",
+  "non-standard": "Non-standard. Check cross-browser support before using.",
```

Comment:
I had a quick look and this should be `nonstandard` without hyphen actually, to be consistent:

```suggestion
  nonstandard: "Non-standard. Check cross-browser support before using.",
```